service/controller/key/key.go

```diff
@@ -95,6 +95,7 @@ const (
 
 const (
 	AnnotationAPIEndpoint       = "kvm-operator.giantswarm.io/api-endpoint"
+	AnnotationComponentVersion  = "kvm-operator.giantswarm.io/component-version"
```

Comment:
Since this annotation isn't used directly but is just the base name for the annotation, what do you think about calling it `AnnotationComponentVersionPrefix` or something? It would make it less surprising when we don't find it on a pod except with a suffix.

Reply:
Updated